app/templates/critter.hbs

```diff
@@ -8,7 +8,7 @@
   </div>
 </div>
 
-<div class="container-fluid mt-4">
+<div class="container-fluid pt-4 pb-4">
```

Comment:
Switched to padding so that there is some space below the shadow size images for scrolling.  Hope that's OK